doc/source/io.rst

```diff
@@ -4093,6 +4093,27 @@ The key functions are:
 
 .. _io.bigquery_reader:
 
+Authentication
+''''''''''''''
+
+You will be authenticated to the specified BigQuery account via Google's Oauth2 mechanism.
+
+Primary auth method is as simple as following the
+prompts in a browser window which will be opened for you. Should the browser not
+be available, or fail to launch, a code will be provided to complete the process
+manually.  Additional information on the authentication mechanism can be found
```

Comment:
@tworec  Can you update the docs to the following text? Please feel free to make changes.

```
Authentication
''''''''''''''

Authentication is possible with either user account credentials or service account credentials. 

Authenticating with user account credentials is as simple as following the prompts in a browser window 
which will be automatically opened for you. You will be authenticated to the specified 
``BigQuery`` account via Google's ``Oauth2`` mechanism. Additional information on the 
authentication mechanism can be found `here <https://developers.google.com/identity/protocols/OAuth2#clientside/>`__.

Authentication with service account credentials is possible via the `'private_key'` parameter. This method
is particularly useful when working on remote servers, where the remote authentication using user account credentials
is not currently supported in Pandas. Additional information on service accounts can be found 
`here <https://developers.google.com/identity/protocols/OAuth2#serviceaccount>`__.


.. note::

The `'private_key'` parameter can be set to either the file path of the service account key in JSON format, or 
key contents of the service account key in JSON format. 

.. note::

A private key can be obtained from the Google developers console by clicking `here <https://console.developers.google.com/permissions/serviceaccounts>`__. Use JSON key type.

```